wire/message.go

```diff
@@ -48,7 +48,8 @@ const (
 	CmdTransactionNotFound
 )
 
-var messageCommandToString = map[MessageCommand]string{
+// MessageCommandToString maps all MessageCommands to their string representation
+var MessageCommandToString = map[MessageCommand]string{
```

Comment:
Consider what you prefer, but I think it's better to export `var NumberOfMessages = len(MessageCommandToString)` rather than `MessageCommandToString`. You can use `init` if needed

Reply:
I don't only need the number, I need the list itself.

I could have created only a list and exported it, but I'd rather have only one datastructure then two, even if I'm over-exporting.